task/thermal/src/bsp/gimlet_b.rs

```diff
@@ -41,7 +40,7 @@ pub(crate) struct Bsp {
     misc_sensors: [TemperatureSensor; NUM_TEMPERATURE_SENSORS],
 
     /// Fans and their respective RPM sensors
-    fans: [(Fan, SensorId); NUM_FANS],
+    fans: [SensorId; NUM_FANS],
```

Comment:
The docstrings above is now out of date, since this is just RPM sensors

Reply:
Fixed